Intersect.Client/Interface/Game/Spells/SpellsWindow.cs

```diff
@@ -61,19 +62,9 @@ public void Update()
             Y = mSpellWindow.Y;
             for (var i = 0; i < Options.MaxPlayerSkills; i++)
             {
-                if (Globals.Me.Spells[i].Id != Guid.Empty)
-                {
-                    Items[i].Pnl.IsHidden = false;
-                    Items[i].Update();
-                    if (Items[i].IsDragging)
-                    {
-                        Items[i].Pnl.IsHidden = true;
-                    }
-                }
-                else
-                {
-                    Items[i].Pnl.IsHidden = true;
-                }
+                var spell = SpellBase.Get(Globals.Me.Spells[i].Id);
+                Items[i].Pnl.IsHidden = spell == null || Items[i].IsDragging;
+                Items[i].Update();
```

Comment:
I don't believe there's a reason to run an update when the spell is blank?

Reply:
oh, you are right, for a moment i though that we needed to updated on those as well in order to properly hide the null ones, just did it so it updates after checking against != null spells and everything seems fine 👍🏽